tools/mkpkg/update_adafruit-libraries

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash -e
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2017-present Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+# set defaults
+GIT_USERNAME=""
+GIT_PASSWORD=""
+
+# Source in GIT_USERNAME and GIT_PASSWORD to avoid API limitations
+[ -f ~/.git.conf ] && source ~/.git.conf
+
+# set http login for curl
+CURL_LOGIN=""
+if [ ! -z "$GIT_USERNAME" -a ! -z "$GIT_PASSWORD" ]; then
+  CURL_LOGIN="--user $GIT_USERNAME:$GIT_PASSWORD"
+fi
+
+# get the realpath to LE Root
+ROOT=$(realpath $(dirname $(realpath $0))/../../)
+cd "$ROOT"
+
+# load adafruit-libraries settings
+. config/options adafruit-libraries
+
+# iterate over all depend packages and try update
+ADAFRUIT_PACKAGES=$PKG_DEPENDS_TARGET
+CHANGED=""
+for PKG in $ADAFRUIT_PACKAGES; do
+  echo -n "check: $PKG... "
+  . config/options $PKG
+  
+  # filter out, all non github packages
+  if [[ $PKG_URL != https://github.com/* ]]; then
+    echo "ignore, not on github"
+    continue
+  fi
+  
+  # get new hash from master branch
+  URL=$(sed "s|\(www\.\)*github.com/|api.github.com/repos/|;s|/archive/.*|/commits/master|" <<< $PKG_URL)
+  GIT_HASH=$(curl --silent --header "Accept: application/vnd.github.VERSION.sha" $CURL_LOGIN $URL)
```

Comment:
`VERSION` should be `v3` rather than the literal VERSION: https://developer.github.com/v3/media/